package.json

```diff
@@ -60,34 +60,34 @@
   },
   "homepage": "https://github.com/mgechev/codelyzer#readme",
   "devDependencies": {
-    "@angular/compiler": "6.0.0-rc.0",
-    "@angular/core": "6.0.0-rc.0",
+    "@angular/compiler": "^6.0.0",
+    "@angular/core": "^6.0.0",
     "@types/chai": "^3.4.33",
-    "@types/less": "0.0.31",
-    "@types/mocha": "^2.2.48",
-    "@types/node": "^6.0.103",
+    "@types/less": "^0.0.32",
+    "@types/mocha": "^5.2.0",
+    "@types/node": "^10.0.4",
     "@types/node-sass": "^3.10.31",
-    "@types/sprintf-js": "0.0.27",
+    "@types/sprintf-js": "^1.1.0",
     "cat": "^0.2.0",
     "chai": "^3.5.0",
```

Comment:
@mgechev Just for reference: I was not able to update `@types/chai`, `chai`, `chai-spies` and `source-map` because this broke some tests.